### DISCUSSION TITLE
文档表格格式显示不正确

### DISCUSSION THREAD

**Discussion by luneShaoGM** [datawhalechina/DOPMC#270] [建议反馈] [answered]
[文章链接](https://datawhalechina.github.io/llm-universe/#/C3/2.%E4%BD%BF%E7%94%A8%20Embedding%20API?id=%e4%b8%80%e3%80%81%e4%bd%bf%e7%94%a8openai-api)
这篇文章第一小节的表格，格式显示不正确

**Comment by logan-zou** [comment, marked as answer]
> [文章链接](https://datawhalechina.github.io/llm-universe/#/C3/2.%E4%BD%BF%E7%94%A8%20Embedding%20API?id=%e4%b8%80%e3%80%81%e4%bd%bf%e7%94%a8openai-api) 这篇文章第一小节的表格，格式显示不正确

已修复，感谢提醒~
